v21.2/sql-statements.md

```diff
@@ -137,11 +137,11 @@ Statement | Usage
 
 Statement | Usage
 ----------|------------
-[`RESET`](reset-vars.html) | Reset a session variable to its default value.
-[`SET`](set-vars.html) | Set a current session variable.
+[`RESET` &lt;session variables&gt;](reset-vars.html) | Reset a session variable to its default value.
+[`SET` &lt;session variables&gt;](set-vars.html) | Set a current session variable.
 [`SET TRANSACTION`](set-transaction.html) | Set the priority for an individual [transaction](transactions.html).
 [`SHOW TRACE FOR SESSION`](show-trace.html) | Return details about how CockroachDB executed a statement or series of statements recorded during a session.
-[`SHOW`](show-vars.html) | List the current session or transaction settings.
+[`SHOW` &lt;session variables&gt;](show-vars.html) | List the current session or transaction settings.
```

Comment:
I think the change is good, and matches our side nav, but the page titles themselves (e.g., at `show-vars.html`) use parentheses rather than `<>`. Should those be amended as well?

Related: For the [style guide](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#placeholders), we landed on using curly brackets `{}` for placeholder values. Would we want to use those here, or do you feel SQL has different conventions (I haven't actually documented much SQL)?

Reply:
This brings us to four implementations:

- no additional, how we found it this morning
- `<>` per left nav
- `()` per site titles
- `{}` per style guide

I will ask in #style-guide.